src/ImageProcessor.Web/Services/RemoteImageService.cs

```diff
@@ -134,7 +135,9 @@ public bool IsValidRequest(string path)
         /// </returns>
         public async Task<byte[]> GetImage(object id)
         {
-            Uri uri = new Uri(id.ToString());
+            Uri uri = new Uri(id.ToString()
```

Comment:
You are right. That would be much better. I try that out within the next days...